system-test/datastore.ts

```diff
@@ -60,6 +60,11 @@ describe('Datastore', () => {
     assert.ok(keys);
   });
 
+  it('should get the project id', async () => {
+    const projectId = await datastore.getProjectId();
+    assert.notEqual(projectId, null);
```

Comment:
It could still come back as `undefined` here, or an empty string :/  What do you think of `assert.ok(projectId)` to capture falsey values?

Reply:
I'm fine with that. I'd just assume that for the other tests to pass the projectid would need to be defined